selene/schema/results/fields.py

```diff
@@ -86,54 +145,103 @@ def resolve_hostname(root, _info):
 
 
 class ResultTask(BaseObjectType):
-    pass
+    """ A task referenced by ID and name """
 
 
-class Result(UserTagsObjectTypeMixin, SimpleEntityObjectType):
+class ResultReport(UUIDObjectTypeMixin, graphene.ObjectType):
+    """ A report referenced by ID """
+
+
+# Override imports Result; importing Override will cause
+# circular imports. Also, gsa does not need all fields
+class ResultOverride(
+    graphene.ObjectType, UUIDObjectTypeMixin, CreationModifactionObjectTypeMixin
+):
+    active = graphene.Boolean()
+    severity = SeverityType()
+    new_severity = SeverityType()
```

Comment:
Not sure if this works or if it needs to be
```python
    severity = graphene.Field(SeverityType)
    new_severity = graphene.Field(SeverityType)
```

Reply:
I think it WORKS, but I'm changing it to the above anyway just because I want to add a docstring to them.